drivers/janitza_umg96rm.json

```diff
@@ -1,13 +1,38 @@
 {
     "common": {
-        "datatype": "float",
         "fncode": 3,
-        "words": 2
+        "words" = 2
     },
     "fields": {
-        "V_L1": {"register": 19000, "description": "Voltage L1-N", "unit": "V"},
-        "V_L2": {"register": 19002, "description": "Voltage L2-N", "unit": "V"},
-        "V_L3": {"register": 19004, "description": "Voltage L3-N", "unit": "V"},
-        "V_L4": {"register": 19004, "description": "Blah", "unit": "V"}
+        "E_exported_Total": {"datatype": "float", "description": "Total export energy", "multiplier": 1, "register": 19076, "unit": "Wh"},
```

Comment:
I see you have `"multiplier": 1` in all fields but it's not really required (unless it's a number different from 1) - so let's remove it.

Reply:
Also `total` should generally be with a small letter (on the next line also)

Reply:
In fact for the energy readings I don't think we need `_total` - just `E_exported` and `E_imported` is probably fine. @osakindoyin what's your take on this?